src/main/java/net/dv8tion/jda/core/events/message/MessageBulkDeleteEvent.java

```diff
@@ -25,9 +25,9 @@
 
 /**
  * Indicates that a bulk deletion is executed in a {@link net.dv8tion.jda.core.entities.TextChannel TextChannel}.
- * <br>Use {@link net.dv8tion.jda.core.JDABuilder#setBulkDeleteSplittingEnabled(boolean)} to true in order to enable this event.
+ * <br>Use {@link net.dv8tion.jda.core.JDABuilder#setBulkDeleteSplittingEnabled(boolean)} to false in order to enable this event.
```

Comment:
"Use ... to false" still sounds incorrect

Reply:
Yes, my bad.